### DISCUSSION TITLE
Idle shutdown time longer than 60 minutes possible?

### DISCUSSION THREAD

**Discussion by theotherdede** [MiczFlor/RPi-Jukebox-RFID#1263] [Q&A]
Hi Phoniebox-Folks,

recently I was wondering if there is a way to set "Idle shutdown" time to more than 60 minutes? All the other shutdown/player related timers will allow values greater than 60 (up to 240 mins).

Looking forward to a solution for this as googling doesn't brought an answer. My little son is using his box heavily, but most of the time his "schedule" doesn't match with that 60 mins timer, so I have to boot the box manually and he has to wait until his favourite music starts playing.

Bests
Dennis

**Comment by lenlennart** [comment]
... did you try setting the value manually (in /home/pi/RPi-Jukebox-RFID/settings/Idle_Time_Before_Shutdown)?
